graphql/database/users/createUser.js

```diff
@@ -51,6 +55,14 @@ const createUser = async (
       ? {
           extensionInstallTimeApprox,
         }
+      : null,
+    v4BetaEnabled
+      ? {
+          backgroundImage: await getRandomBackgroundImage(
```

Comment:
I believe using `await` in a property value will cause assigning `undefined` to `userInfo` here, which is synchronous. That's probably the cause of the failing tests.